kong.conf.default

```diff
@@ -1989,3 +1989,88 @@
                         # * No recursion is performed. Only .wasm files at the
                         #   top level are registered
                         # * This path _may_ be a symlink to a directory.
+
+#------------------------------------------------------------------------------
+# WASM injected directives
```

Comment:
```suggestion
# Wasm injected directives
```

(I think I'm going to die on that hill :laughing: )

Reply:
All-caps was chosen for consistency with the other section headers (elsewhere we have `GENERAL` and `NGINX injected directives`). That said I don't particularly mind changing it :shrug: